.github/pr_management/common.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * Collection of common functions to use in this folder
+ */
+import * as core from '@actions/core'
+import * as github from '@actions/github';
+const token = core.getInput("repo-token");
+const octokit = github.getOctokit(token);
+
+const owner = github.context.repo.owner;
+const repo = github.context.repo.repo;
+const issue_number = github.context.issue.number;
+const actor = github.context.actor;
+
+//// variables to configure
+// todo change in teammates
+const usualTimeForChecksToRun = 5000; // 20 * 60 * 1000; // min * sec * ms
+
+// to prevent cyclical checking when checking for passing runs. note: needs to match names assigned by workflow files
+const excludedChecksNames = {
+    "Handle PR that may be draft": 1,
+    "Handle PR that may be ready for review": 1,
+};
+
+export async function wereReviewCommentsAdded(pr, sinceTimeStamp: string) {
+    isValidTimestamp(sinceTimeStamp);
+
+    const comments = await octokit.rest.pulls
+        .listReviewComments({
+            owner,
+            repo,
+            pull_number: pr.number,
+            since: sinceTimeStamp, // todo unsure if this works as expected --> test
+        })
+        .then(res => {
+            core.info("these comments were retrieved\n" + res);
+            return res;
+        })
+        .catch(err => {
+            throw err;
+        });
+
+    return comments.data.length > 0;
+}
+
+function isValidTimestamp(sinceTimeStamp: string) {
```

Comment:
I think this should return a boolean, or u can change the function name

Reply:
Thank you, changing it to validateTimestamp